configs/policy.yaml

```diff
@@ -1823,6 +1824,163 @@ acct_settings:
       subsidize_affordable: True
       enable_in_scenarios: []
 
+  # office lump_sum_accounts
+  office_lump_sum_accounts:
+    dublin_office_bond_settings:
+      name: "Dublin Office Subsidy Bond"
+      sending_buildings_subaccount_def: regional
+      total_amount: 24675325
+      receiving_buildings_filter: juris == 'Dublin' and (
+                                    tra_id == 'tra1' or
+                                    tra_id == 'tra2a' or
+                                    tra_id == 'tra2b' or
+                                    tra_id == 'tra2c')
+      enable_in_scenarios: ["24"]
```

Comment:
I'm guessing `subsidize_affordable: True` is not needed for commercial. I'll check it out.

Reply:
I think so, because unlike the residential model where "subsidized_affordable: True" is activated to calculate subsidized_units as a subset of all res_units in the building, this model is counting the sqft of the entire building.

Reply:
Okay got it, bc it becomes "create_deed_restricted", which creates the subsidized units. It does seem a bit harder to tease out "subsidized sqft" in the office model. And frankly, I'm not sure it makes much sense to. I like these models as subsidies that make projects feasible, and I think you could say they're subsidized or we're subsidizing some inclusionary aspect. Designating parts of them as "affordable" is tricky, even for residential where we're doing it so that we can say a couple of units are 100% subsidized.

Reply:
Yes, because there is really no such a thing like 'affordable office space' in real world. Basically, without this subsidy, the entire building wouldn't have existed.  
Similarly, when we talk about residential lump sum, if a 100-unit subsidized building has 10 subsidized units and 10 inclusionary units, we can say "the subsidy created 10 subsidized units",  but the other 80 market-rate units and 10 inclusionary units wouldn't have been created without the money. So, with our current model, subsidies not only create DR, but also enable non-DR.